### ISSUE TITLE
【EESS】New submissions for Mon, 20 Nov 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#878]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### End-to-end autoencoding architecture for the simultaneous generation of  medical images and corresponding segmentation masks
 - **Authors:** Aghiles Kebaili, Jérôme Lapuyade-Lahorgue, Pierre Vera, Su Ruan
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2311.10472
 - **Pdf link:** https://arxiv.org/pdf/2311.10472
 - **Abstract**
 Despite the increasing use of deep learning in medical image segmentation, acquiring sufficient training data remains a challenge in the medical field. In response, data augmentation techniques have been proposed; however, the generation of diverse and realistic medical images and their corresponding masks remains a difficult task, especially when working with insufficient training sets. To address these limitations, we present an end-to-end architecture based on the Hamiltonian Variational Autoencoder (HVAE). This approach yields an improved posterior distribution approximation compared to traditional Variational Autoencoders (VAE), resulting in higher image generation quality. Our method outperforms generative adversarial architectures under data-scarce conditions, showcasing enhancements in image quality and precise tumor mask synthesis. We conduct experiments on two publicly available datasets, MICCAI's Brain Tumor Segmentation Challenge (BRATS), and Head and Neck Tumor Segmentation Challenge (HECKTOR), demonstrating the effectiveness of our method on different medical imaging modalities.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
### Clustering Techniques for Stable Linear Dynamical Systems with  applications to Hard Disk Drives
 - **Authors:** Nikhil Potu Surya Prakash, Joohwan Seo, Jongeun Choi, Roberto Horowitz
 - **Subjects:** Systems and Control (eess.SY); Artificial Intelligence (cs.AI); Machine Learning (cs.LG); Dynamical Systems (math.DS); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2311.10322
 - **Pdf link:** https://arxiv.org/pdf/2311.10322
 - **Abstract**
 In Robust Control and Data Driven Robust Control design methodologies, multiple plant transfer functions or a family of transfer functions are considered and a common controller is designed such that all the plants that fall into this family are stabilized. Though the plants are stabilized, the controller might be sub-optimal for each of the plants when the variations in the plants are large. This paper presents a way of clustering stable linear dynamical systems for the design of robust controllers within each of the clusters such that the controllers are optimal for each of the clusters. First a k-medoids algorithm for hard clustering will be presented for stable Linear Time Invariant (LTI) systems and then a Gaussian Mixture Models (GMM) clustering for a special class of LTI systems, common for Hard Disk Drive plants, will be presented.
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### User Dynamics-Aware Edge Caching and Computing for Mobile Virtual  Reality
 - **Authors:** Mushu Li, Jie Gao, Conghao Zhou, Xuemin Shen, Weihua Zhuang
 - **Subjects:** Signal Processing (eess.SP); Multimedia (cs.MM); Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2311.10645
 - **Pdf link:** https://arxiv.org/pdf/2311.10645
 - **Abstract**
 In this paper, we present a novel content caching and delivery approach for mobile virtual reality (VR) video streaming. The proposed approach aims to maximize VR video streaming performance, i.e., minimizing video frame missing rate, by proactively caching popular VR video chunks and adaptively scheduling computing resources at an edge server based on user and network dynamics. First, we design a scalable content placement scheme for deciding which video chunks to cache at the edge server based on tradeoffs between computing and caching resource consumption. Second, we propose a machine learning-assisted VR video delivery scheme, which allocates computing resources at the edge server to satisfy video delivery requests from multiple VR headsets. A Whittle index-based method is adopted to reduce the video frame missing rate by identifying network and user dynamics with low signaling overhead. Simulation results demonstrate that the proposed approach can significantly improve VR video streaming performance over conventional caching and computing resource scheduling strategies.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
### UniMOS: A Universal Framework For Multi-Organ Segmentation Over  Label-Constrained Datasets
 - **Authors:** Can Li, Sheng Shao, Junyi Qu, Shuchao Pang, Mehmet A. Orgun
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2311.10251
 - **Pdf link:** https://arxiv.org/pdf/2311.10251
 - **Abstract**
 Machine learning models for medical images can help physicians diagnose and manage diseases. However, due to the fact that medical image annotation requires a great deal of manpower and expertise, as well as the fact that clinical departments perform image annotation based on task orientation, there is the problem of having fewer medical image annotation data with more unlabeled data and having many datasets that annotate only a single organ. In this paper, we present UniMOS, the first universal framework for achieving the utilization of fully and partially labeled images as well as unlabeled images. Specifically, we construct a Multi-Organ Segmentation (MOS) module over fully/partially labeled data as the basenet and designed a new target adaptive loss. Furthermore, we incorporate a semi-supervised training module that combines consistent regularization and pseudolabeling techniques on unlabeled data, which significantly improves the segmentation of unlabeled data. Experiments show that the framework exhibits excellent performance in several medical image segmentation tasks compared to other advanced methods, and also significantly improves data utilization and reduces annotation cost. Code and models are available at: https://github.com/lw8807001/UniMOS.
## Keyword: multi organ
There is no result 
### K-space Cold Diffusion: Learning to Reconstruct Accelerated MRI without  Noise
 - **Authors:** Guoyao Shen, Mengyu Li, Chad W. Farris, Stephan Anderson, Xin Zhang
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG); Medical Physics (physics.med-ph)
 - **Arxiv link:** https://arxiv.org/abs/2311.10162
 - **Pdf link:** https://arxiv.org/pdf/2311.10162
 - **Abstract**
 Deep learning-based MRI reconstruction models have achieved superior performance these days. Most recently, diffusion models have shown remarkable performance in image generation, in-painting, super-resolution, image editing and more. As a generalized diffusion model, cold diffusion further broadens the scope and considers models built around arbitrary image transformations such as blurring, down-sampling, etc. In this paper, we propose a k-space cold diffusion model that performs image degradation and restoration in k-space without the need for Gaussian noise. We provide comparisons with multiple deep learning-based MRI reconstruction models and perform tests on a well-known large open-source MRI dataset. Our results show that this novel way of performing degradation can generate high-quality reconstruction images for accelerated MRI.
### Joint channel estimation and data detection in massive MIMO systems  based on diffusion models
 - **Authors:** Nicolas Zilberstein, Ananthram Swami, Santiago Segarra
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2311.10311
 - **Pdf link:** https://arxiv.org/pdf/2311.10311
 - **Abstract**
 We propose a joint channel estimation and data detection algorithm for massive multilple-input multiple-output systems based on diffusion models. Our proposed method solves the blind inverse problem by sampling from the joint posterior distribution of the symbols and channels and computing an approximate maximum a posteriori estimation. To achieve this, we construct a diffusion process that models the joint distribution of the channels and symbols given noisy observations, and then run the reverse process to generate the samples. A unique contribution of the algorithm is to include the discrete prior distribution of the symbols and a learned prior for the channels. Indeed, this is key as it allows a more efficient exploration of the joint search space and, therefore, enhances the sampling process. Through numerical experiments, we demonstrate that our method yields a lower normalized mean squared error than competing approaches and reduces the pilot overhead.
### Phase Guided Light Field for Spatial-Depth High Resolution 3D Imaging
 - **Authors:** Geyou Zhang, Ce Zhu, Kai Liu, Yipeng Liu
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2311.10568
 - **Pdf link:** https://arxiv.org/pdf/2311.10568
 - **Abstract**
 On 3D imaging, light field cameras typically are of single shot, and however, they heavily suffer from low spatial resolution and depth accuracy. In this paper, by employing an optical projector to project a group of single high-frequency phase-shifted sinusoid patterns, we propose a phase guided light field algorithm to significantly improve both the spatial and depth resolutions for off-the-shelf light field cameras. First, for correcting the axial aberrations caused by the main lens of our light field camera, we propose a deformed cone model to calibrate our structured light field system. Second, over wrapped phases computed from patterned images, we propose a stereo matching algorithm, i.e. phase guided sum of absolute difference, to robustly obtain the correspondence for each pair of neighbored two lenslets. Finally, by introducing a virtual camera according to the basic geometrical optics of light field imaging, we propose a reorganization strategy to reconstruct 3D point clouds with spatial-depth high resolution. Experimental results show that, compared with the state-of-the-art active light field methods, the proposed reconstructs 3D point clouds with a spatial resolution of 1280$\times$720 with factors 10$\times$ increased, while maintaining the same high depth resolution and needing merely a single group of high-frequency patterns.
### Laser-Based Fabrication of Microstructures on Nickel Thin Films and Its  Applications in On-Chip Thin Film Inductors
 - **Authors:** Srikanth Itapu, Vamsi Borra, Daniel G. Georgiev
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2311.10660
 - **Pdf link:** https://arxiv.org/pdf/2311.10660
 - **Abstract**
 This work reports on the fabrication of microbump structures on Ni films by single-pulse, localized laser irradiation. Conditions for the reproducible formation of such microstructures have been identified in terms of laser-irradiation and film parameters after systematic studies involving a relevant parameter space. The cracks and voids morphology of the sputtered films was rendered undesirable and hence, smoother Ni thin film of same thickness (200nm) were deposited by vacuum evaporation. The continuous nature of the film resulted in radially symmetric thermal expansion and deformation, thus achieving a high yield of microstructures. An improvement in the inductance and the quality factor of on-chip spiral inductors incorporating such laser-microstructured ferromagnetic nickel thin films was observed, which demonstrates the potential of such a laser-based method for fabrication or fine tuning of various micro-/nanoelectric/electronic sensor and other components and systems.
